content/en/APIUseCases/create_asset.md

```diff
@@ -210,6 +210,7 @@ curl -X PUT 'https://api.cobalt.io/assets/{{% asset-id %}}' \
           }' \
   -v
 ```
+<!-- Remove asset size and coverage when Comprehensive Pentests are launched (Sept 7). -->
```

Comment:
While you're right to add this note, I see no evidence that our API docs are changing to remove "size" and "coverage", ref https://github.com/cobalthq/cobalt-api-docs/pulls